### PR TITLE
Move product selection to welcome

### PR DESCRIPTION
**Pull request by imobachgs**
Requires https://github.com/yast/yast-yast2/pull/619, https://github.com/yast/yast-packager/pull/273 and
https://github.com/yast/yast-country/pull/138.

Move product's selection to welcome screen.

complex-welcome-license

### CODE REVIEW

File: src/lib/installation/clients/inst_complex_welcome.rb [hunk 1]
```diff
@@ -22,333 +22,108 @@
...
+
+      when :next
+        return if Mode.config
+        return unless Language.CheckIncompleteTranslation(Language.language)
+        if selected_product.nil?
+          Yast::Popup.Error(_("Please select a product to install."))
+          return nil
+        end
+        setup_final_choice
+        merge_and_run_workflow if selected_product
```
Comment (teclator) [inline src/lib/installation/clients/inst_complex_welcome.rb]: is this check really needed?

File: src/lib/installation/clients/inst_complex_welcome.rb [hunk 2]
```diff
@@ -22,333 +22,108 @@
...
-      # FIXME: remove the hack, please
-      if @language == "nn_NO"
-        log.info "Nynorsk not translated, using Bokm\u00E5l"
-        Language.WfmSetGivenLanguage("nb_NO")
-      else
-        Language.WfmSetLanguage
-      end
-    end
-
+    # Most of the work is done by #retranslate_yast. If changing to english if
```
Comment (teclator) [inline src/lib/installation/clients/inst_complex_welcome.rb]: typo? is needed?

File: src/lib/installation/clients/inst_complex_welcome.rb (hunk 1 above)
Comment (imobachgs) [reply src/lib/installation/clients/inst_complex_welcome.rb]: No, it is not :)

File: test/inst_complex_welcome_test.rb [hunk 3]
```diff
@@ -48,6 +67,12 @@
 
   describe "#main" do
     let(:restarting) { false }
```
Comment (teclator) [inline test/inst_complex_welcome_test.rb]: Please, remove also this from the test as it is not needed anymore

**Comment by teclator** [comment]
Rubocop is still unhappy

**Comment by imobachgs** [comment]
Yes, you are right. I did not check Travis because I expected a dependencies problem (new yast2-packager and yast2 are not available yet). Now it is failing due to missing dependencies :)

**Comment by teclator** [comment]
Thnx ;) it LGTM

**Pull request merged**